docs/modules/release-notes/pages/all-releases/4_14.adoc

```diff
@@ -0,0 +1,117 @@
+
+* Appium Inspector
+
+image:session-explorer-obfuscate-private-data-inspector.png[width=500,alt="The obfuscated data in the Appium Inspector section"]
+
+
+== Auto-upgrade deviceConnect: force offline devices if deviceConnect is 2 major versions behind the latest
+
+To ensure Cloud and Hybrid customers can always enjoy the latest features, we added a new policy to force devices offline on the Kobiton portal if deviceConnect on the host machine is 2 major versions behind the latest.
+
+This means for the v4.14 release, all devices hosted by deviceConnect v4.12 or below will become offline. Devices hosted by deviceConnect v4.13 are still online until v4.15 is released (if not updated).
+
+[WARNING]
+There is a known issue that if a new hosting machine with deviceConnect v4.12 or below is added to an organization , the hosting machine and all its devices are not displayed in the Device Management page in the portal. Therefore, make sure you *install deviceConnect v4.13 or later* to new hosting machines.
+
+When devices are offline due to deviceConnect version being too old, you will see  `Kobiton upgrade required` under the device message in *Settings* -> *Device Management*:
+
+image:force-offline-devices-old-deviceconnect.png[width=1000,alt="The Kobiton upgrade required message under Device Management"]
+
+When this happens, if automatic upgrade is enabled in your organization, the system will download the latest version and update automatically. If this is turned off, you can manually upgrade deviceConnect on the host machine by choosing *Upgrade*.
+
+image:upgrade-host-machine-device-management.png[width=1000,alt="The Upgrade button next to the hosting machine name in Device Management"]
+
+== Minor improvements and bug fixes
+
+This release includes several enhancements and bug fixes to improve your day-to-day testing.
+
+* Addressed the following Script-based test automation and Device Inspector items:
+
+** Added support for JSON strings passed as parameters.
+** Fixed an issue with uploading test runner to the Portal.
+** Fixed a bug that prevents image injection via app instrumentation in Appium.
+** Implemented method `appium:optionalIntentArguments` .
+** The `sendKeys` action now works in generated C# Appium script.
+** Fixed an issue with `childSelector` in `UIAutomator` only returns 1 element when there is a list of elements.
+
+* Other items addressed:
+
+** Videos downloaded from iOS sessions can now be opened by Quicktime and Windows Media Player.
+** Exported HAR files (Network Payload Capture) can now be extracted in Windows without downloading 3rd-party apps.
+** Fixed an issue with devices stuck in Utlizing on the Portal if multiple devices come online at the same time in organizations with no Cleanup Policy.
+** Improve FPS (frames per second) in Manual sessions with Standard mode (up to 10 FPS).
```

Comment:
remove "(up to 10 FPS)" - we should not publicize that.

Reply:
removed